README.md

```diff
@@ -33,3 +33,7 @@ See the readme file in the ```./portable``` directory for more information.
 ### Code Formatting
 FreeRTOS files are formatted using the "uncrustify" tool. The configuration file used by uncrustify can be found in the [FreeRTOS/FreeRTOS repository](https://github.com/FreeRTOS/FreeRTOS/blob/master/tools/uncrustify.cfg). 
 
+### Spelling
+*lexicon.txt* contains words that are not traditionally found in an English dictionary. It is used by the spellchecker to verify the various jargon, variable names, and other odd words used in the FreeRTOS code base. If your pull request fails to pass the spelling check fix the error. If it is a variable name or the like that IS spelled correctly, but unknown to the lexicon, then add the word to *lexicon.txt*. 
```

Comment:
If your pull request fails to pass the spelling check fix the error.  odd sentence

Reply:
Hmm true, must have been tired :) I will fix this